wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/wsworkunits/WsWorkunitsClientSoapProxyWrapper.java

```diff
@@ -0,0 +1,105 @@
+package org.hpccsystems.ws.client.wrappers.wsworkunits;
+
+import org.apache.axis.client.Stub;
+import org.apache.commons.lang3.NotImplementedException;
+import org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ArrayOfEspException;
+import org.hpccsystems.ws.client.gen.wsworkunits.v1_73.EspException;
+import org.hpccsystems.ws.client.gen.wsworkunits.v1_73.WsWorkunitsServiceSoap;
+import org.hpccsystems.ws.client.gen.wsworkunits.v1_73.WsWorkunitsServiceSoapProxy;
+import org.hpccsystems.ws.client.platform.Version;
+import org.hpccsystems.ws.client.platform.WorkunitInfo;
+import org.hpccsystems.ws.client.utils.Connection;
+
+public class WsWorkunitsClientSoapProxyWrapper
+{
+    
+    private org.hpccsystems.ws.client.gen.wsworkunits.v1_56.WsWorkunitsServiceSoap version5WorkunitsServiceSoap = null;
+    private org.hpccsystems.ws.client.gen.wsworkunits.v1_69.WsWorkunitsServiceSoap version6WorkunitsServiceSoap = null;
+    private org.hpccsystems.ws.client.gen.wsworkunits.v1_73.WsWorkunitsServiceSoap version7WorkunitsServiceSoap = null;
+
+    private org.hpccsystems.ws.client.gen.wsworkunits.v1_56.WsWorkunitsServiceSoapProxy version5WorkunitsServiceSoapProxy = null;

```

Comment:
version1_5WorkunitsServiceSoapProxy, version1_6WorkunitsServiceSoapProxy , and version1_7WorkunitsServiceSoapProxy, since at some point we could have a 2_5, 2_6, and 2_7 versioned wsworkunits

Reply:
this comment still applies